### PR TITLE
Adding new app

### PR DESCRIPTION
**Pull request by zoltan-baba**
Changed the Adding a new app section to reflect how the process works now.

### CODE REVIEW

File: docs/adding-a-new-app/setting-up-configuration.md [hunk 1]
```diff
@@ -6,26 +6,26 @@ Currently Bitrise will detect `iOS`, `Android`, `Xamarin` and `fastlane` project
 You can configure other types of projects too, but that will require manual
 configuration.
 
+## Automatic project configuration
 
-## Project configuration with automatic repository scanner
+Enter the name of the default branch of your repository where your project is located. This branch should include the configuration of your project. Once you hit `Next`, Bitrise will automatically start the validation of the repository.
+
+  ![Choosing branch](/img/adding-a-new-app/choose-branch.png)
```
Comment (slapec93) [inline docs/adding-a-new-app/setting-up-configuration.md]: Can you please add this picture to the PR? Or will it be available some other way?

File: docs/adding-a-new-app/setting-up-configuration.md [hunk 2]
```diff
@@ -6,26 +6,26 @@ Currently Bitrise will detect `iOS`, `Android`, `Xamarin` and `fastlane` project
...
 
-After successful validation Bitrise will scan your repository and give you a default workflow
+If the validation is successful, Bitrise will scan your repository and give you a default workflow based on the configuration of the project.
+
+## Manual project configuration
 
+If the validation fails, choose the `Restart scanning without validation` option.
 
-## Skipping the scanner - full manual configuration
+  ![Validation failed](/img/adding-a-new-app/validation-failed.png)
```
Comment (slapec93) [inline docs/adding-a-new-app/setting-up-configuration.md]: Same question here

File: docs/adding-a-new-app/setting-up-configuration.md [hunk 3]
```diff
@@ -6,26 +6,26 @@ Currently Bitrise will detect `iOS`, `Android`, `Xamarin` and `fastlane` project
...
 
-If for some reason you want to skip the scanner, you can choose
-the **Configure manually without project scanning** option,
-in the **Validation setup** section.
+In this case, you have to configure the project manually. Click `Next`. You will see the `Validating Repository` message again but this time Bitrise only checks that we have access to the specified repository.
 
-![Configure manually without project scanning](/img/adding-a-new-app/validation_configure_manually.png)
+Choose the project type (for example, Xamarin) and specify the necessary inputs (for example, the path to the Xamarin Solution file). You can also select the stack on which you wish to run your builds.
```
Comment (slapec93) [inline docs/adding-a-new-app/setting-up-configuration.md]: Shouldn't we add some screenshot about this?

File: docs/adding-a-new-app/setting-up-configuration.md (hunk 1 above)
Comment (zoltan-baba) [reply docs/adding-a-new-app/setting-up-configuration.md]: I forgot to add the screenshots to the branch I pushed to remote... sorry :) Added them now.

File: docs/adding-a-new-app/setting-up-configuration.md (hunk 3 above)
Comment (zoltan-baba) [reply docs/adding-a-new-app/setting-up-configuration.md]: You're absolutely right, I've added one.

File: docs/adding-a-new-app/setting-up-configuration.md (hunk 2 above)
Comment (zoltan-baba) [reply docs/adding-a-new-app/setting-up-configuration.md]: see above :)

**Pull request merged**